src/components/attendee/Button/Button.tsx

```diff
@@ -25,6 +25,7 @@ export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   border?: BorderVariant;
   unrounded?: boolean;
   marginless?: boolean;
+  large?: boolean;
```

Comment:
[NOTE][OOS]
There is a way of doing this in line with how `  CN["border-" + border],`  works.

I think it is highly likely we'd need more sizes in which case we can update this prop to use `SizeProp` like the experiment in https://github.com/sparkletown/sparkle/blob/redesign-2021/src/components/attendee/Spacer/Spacer.tsx

The style file in it has this idea realized as: 
```scss
$-sizes: (
  "none": 0,
  //
  "min": space.gap(1),
  "max": space.gap(10),
  //
  "small": space.gap(2.5),
  "medium": space.gap(5),
  "large": space.gap(10),
);
```